braph2genesis/graphs/_OrdMxBUT.gen.m

```diff
@@ -88,7 +90,7 @@ those of binary undirected (BU) multiplex graphs derived from the same
 value = Graph.NONNEGATIVE * ones(layernumber);
 
 %%% ¡prop!
-A (result, cell) is the cell array containing the multiplex binary adjacency matrices of the binary undirected multiplex. 
+A (result, cell) is the cell array containing the multiplex binary adjacency matrices of the binary undirected multiplex with fixed thresholds (BUT) and the ordinal connections between layers.
```

Comment:
A (result, cell) is the cell array containing the binary supra-adjacency matrix of the binary undirected multiplex with fixed thresholds (BUT).